geomstats/geometry/matrices.py

```diff
@@ -48,8 +57,7 @@ def equal(mat_a, mat_b, atol=TOLERANCE):
 
     @staticmethod
     def mul(*args):
-        """
-        Return the product of matrices a1, ..., an.
+        """Calculate the product of matrices a1, ..., an.
```

Comment:
Should we use "Calculate"/ "Compute" to describe the functions? I had deleted them all in my PR, to save space and also bcs I thought it was not essential, but happy to put them back everywhere.